invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -12,20 +12,47 @@
         },
         "access" : {
           "properties" : {
-            "access_right" : {
+            "record" : {
               "type" : "keyword"
             },
-            "created_by" : {
-              "type" : "long"
+            "files" : {
+              "type" : "keyword"
+            },
+            "owned_by" : {
+              "properties": {
+                "user": {
+                  "type": "keyword"
```

Comment:
partially yes (although the system user isn't strictly a user and thus would likely require another property) and because i thought there would be no need for any other type of query than exact matches anyways

primarily, i chose the string value `"system"` for the system user's ID because i wasn't sure if it was 100% guaranteed that user IDs start at 1 in every case (e.g. other DBs?), and i found `"system"` more memorable than negative integer values

Reply:
Comment: The system identity is a special identity and not a user. The reason for giving an id to the system identity is because an identity must have an id, but the system identity does not provide the ``UserNeed(system_id)``. Because the system identity is not a user, it also cannot own a record.

Reply:
Thanks for clarifying!